examples/jit-quantized/resnet.py

```diff
@@ -0,0 +1,22 @@
+import torch
+import torchvision
+
+
+def save_resnet18_script_module(
+    filename: str, pretrained: True, quantize: False
```

Comment:
I'm not very familiar with python type annotations, but do True/False make sense in this setting or should these be `bool` (or default values, e.g. `pretrained=True`.

Reply:
woops my bad, updated